component/tests/golden/vshn/appcat/appcat/21_composition_vshn_postgresrestore.yaml

```diff
@@ -1103,83 +1099,23 @@ spec:
                         for: 1h
                         labels:
                           severity: warning
-                  - name: postgresql-memory
```

Comment:
Can you make sure nothing is changed for PostgreSQL? You introduced a generic function but I expect no changes for existing PostgreSQL code.

Reply:
I just made it the closest possible to original